src/posthog-core.ts

```diff
@@ -997,7 +997,7 @@ export class PostHog {
      * @param {Function} [callback] The callback function will be called once the feature flags are ready or when they are updated.
      *                              It'll return a list of feature flags enabled for the user.
      */
-    onFeatureFlags(callback: (flags: string[], variants: Record<string, boolean | string>) => void): void {
+    onFeatureFlags(callback: (flags: string[], variants: Record<string, JsonType>) => void): void {
```

Comment:
variants can't be jsontype, can they?

Reply:
good catch this was bad typing while thinking i should pass through payloads